service/src/test/java/bio/terra/pipelines/configuration/internal/ImputationConfigurationTest.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.pipelines.configuration.internal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import bio.terra.pipelines.app.configuration.internal.ImputationConfiguration;
+import bio.terra.pipelines.testutils.BaseEmbeddedDbTest;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class ImputationConfigurationTest extends BaseEmbeddedDbTest {
```

Comment:
yeah it's because we're using @Autowired ... all our configuration tests use `BaseEmbeddedDbTest`. we could create a `BaseSpringBootTest` but we'd still need to configure a db setup in that case, so not sure it's worth it